src/Apps/Artist/Routes/Consign/__tests__/ConsignRoute.test.tsx

```diff
@@ -0,0 +1,194 @@
+import { ConsignRouteFixture } from "Apps/__tests__/Fixtures/Artist/Routes/ConsignRouteFixture"
+import { SystemContextProvider } from "Artsy"
+import { renderRelayTree } from "DevTools"
+import React from "react"
+import { graphql } from "relay-runtime"
+import { ConsignRouteFragmentContainer } from "../index"
+
+import { ConsignRoute_Test_QueryRawResponse } from "__generated__/ConsignRoute_Test_Query.graphql"
+
+jest.unmock("react-relay")
+
+jest.mock("Apps/Artist/Routes/Consign/Utils/getConsignmentData", () => ({
+  getConsignmentData: () => {
+    const {
+      artistConsignmentFixture,
+    } = require("Apps/__tests__/Fixtures/Artist/Routes/ConsignRouteFixture")
+
+    return artistConsignmentFixture
+  },
+}))
+
+describe("ConsignRoute", () => {
+  const getWrapper = async (
+    response: ConsignRoute_Test_QueryRawResponse = ConsignRouteFixture
+  ) => {
+    const match: any = {
+      location: {
+        pathname: "/artist/alex-katz/consign",
+      },
+      params: {
+        artistID: "alex-katz",
+      },
+    }
+    const router: any = {
+      replace: jest.fn(),
+    }
+
+    return await renderRelayTree({
+      Component: ({ artist, artworksByInternalID }) => {
+        return (
+          <SystemContextProvider user={{ type: "Admin" }}>
+            <ConsignRouteFragmentContainer
+              artist={artist}
+              artworksByInternalID={artworksByInternalID}
+              match={match}
+              router={router}
+            />
+          </SystemContextProvider>
+        )
+      },
+      query: graphql`
+        query ConsignRoute_Test_Query(
+          $artistID: String!
+          $recentlySoldArtworkIDs: [String]!
+        ) @raw_response_type {
```

Comment:
See [@raw_response_type](https://mrtnzlml.com/docs/relay/directives#raw_response_type) for full explanation, but the TLDR is: 

If you have GraphQL query `A` that contains a nested fragment query `B`, importing the generated type for `A` would only contain a `$ref` to `B` due to how relay implements data masking. By using `@raw_response_type` it will spit out the whole deal, which makes it ideal for typing raw JSON data output.